homu/main.py

```diff
@@ -274,7 +274,11 @@ class AuthState(IntEnum):
     NONE = 1
 
 
-def verify_auth(username, repo_cfg, state, auth, realtime):
+def verify_auth(username, repo_cfg, state, auth, realtime, my_username):
+    # homu does talk to itself in comments
```

Comment:
So I was looking at an example and was *really* confused as to how this was happening...I deduced it must be through the 📌 approval message, but it took me a while to see:

```
state.add_comment(':pushpin: Commit {:.7} has been approved by `{}`\n\n<!-- @{} r={} {} -->'.format(state.head_sha, approver, my_username, approver, state.head_sha))
```

This hidden comment recursing into the code above again...it's like recursively evil.

Can you just tweak your (python) comment to clarify that the comment it's talking about is a markdown comment in the github comment, something like:

```
# In some cases (e.g. r+) we recursively talk to ourself via a hidden markdown comment in the message
```